migrator/migration.py

```diff
@@ -133,6 +134,15 @@ def disable_migration_service_plan(self):
         for service_plan_visibility_id in self.service_plan_visibility_ids:
             cf.disable_plan_for_org(service_plan_visibility_id, self.client)
 
+    def create_bare_migrator_instance_in_org_spaces(self):
+        for space_id in cf.get_all_space_ids_for_org(self.org_id, self.client):
```

Comment:
why are we doing this in all spaces?

Reply:
D'oh!  I transposed our work on the previous PR to this feature, and here we are.  I will adjust to focus on just the single space tied to the instance of the `Migrator` object, thanks for catching this!

I am so ready for the pandemic 🧠 🌫️ to be a thing of the past...

Reply:
Fixed; this wasn't even correct in the first place because it was referencing `self.space_id` in the loop...